src/components/settings/SettingsSSOTab/steps/SSOConfigConfigureStep.jsx

```diff
@@ -135,121 +210,279 @@ const SSOConfigConfigureStep = ({
+          <Form.Checkbox checked={isUsingSap} onChange={handleChange}>
+            I am using SAP Success Factors as an Identity Provider
+          </Form.Checkbox>
         </Form.Group>
 
-        <Form.Group>
-          <Form.Control
-            type="text"
-            onChange={(e) => {
-              addConfigVal('attr_user_permanent_id', e.target.value);
-              setFormUpdated(true);
-            }}
-            maxLength={128}
-            floatingLabel="User ID Attribute"
-            defaultValue={(connectError ? errorData.userId : existingConfigData?.attr_user_permanent_id)}
-          />
-          <Form.Text>
-            URN of the SAML attribute that we can use as a unique, persistent user ID. Leave blank for default.
-          </Form.Text>
-        </Form.Group>
-        <Form.Group>
-          <Form.Control
-            type="text"
-            onChange={(e) => {
-              addConfigVal('attr_full_name', e.target.value);
-              setFormUpdated(true);
-            }}
-            maxLength={255}
-            floatingLabel="Full Name Attribute"
-            defaultValue={(connectError ? errorData.fullName : existingConfigData?.attr_full_name)}
-          />
-          <Form.Text>
-            URN of SAML attribute containing the user&apos;s full name. Leave blank for default.
-          </Form.Text>
-        </Form.Group>
-        <Form.Group>
-          <Form.Control
-            type="text"
-            onChange={(e) => {
-              addConfigVal('attr_first_name', e.target.value);
-              setFormUpdated(true);
-            }}
-            maxLength={128}
-            floatingLabel="First Name Attribute"
-            defaultValue={(connectError ? errorData.firstName : existingConfigData?.attr_first_name)}
-          />
-          <Form.Text>
-            URN of SAML attribute containing the user&apos;s first name. Leave blank for default.
-          </Form.Text>
-        </Form.Group>
-        <Form.Group>
-          <Form.Control
-            type="text"
-            onChange={(e) => {
-              addConfigVal('attr_last_name', e.target.value);
-              setFormUpdated(true);
-            }}
-            maxLength={128}
-            floatingLabel="Last Name Attribute"
-            defaultValue={(connectError ? errorData.lastName : existingConfigData?.attr_last_name)}
-          />
-          <Form.Text>
-            URN of SAML attribute containing the user&apos;s last name. Leave blank for default.
-          </Form.Text>
-        </Form.Group>
+        {!isUsingSap && (
+        <>
+          <Form.Group>
+            <Form.Control
+              type="text"
+              isInvalid={!nameValid}
+              onChange={(e) => {
+                validateField('name', e.target.value);
+              }}
+              floatingLabel="SSO Configuration Name"
+              defaultValue={(connectError ? errorData.displayName : existingConfigData?.display_name)}
```

Comment:
Does the errorData suggested values need to change for these new ones? If we get an error and it has to get repopulated?

Reply:
https://github.com/openedx/frontend-app-admin-portal/blob/a61c9849a1011110453ff855036d3d88c16a7f52/src/components/settings/SettingsSSOTab/steps/SSOConfigConfigureStep.jsx#L56

Reply:
🤔